OpenRA.Mods.Common/Widgets/Logic/SettingsLogic.cs

```diff
@@ -716,6 +720,29 @@ static bool ShowStatusBarsDropdown(DropDownButtonWidget dropdown, GameSettings s
 			return true;
 		}
 
+		static bool ShowTargetLinesDropdown(DropDownButtonWidget dropdown, GameSettings s)
```

Comment:
Probably outside this scope but the return values for this method is never used, this is also true for all *Dropdown methods.

Reply:
Outside scope indeed, but added a commit to fix this.